CHANGELOG

```diff
@@ -1,4 +1,6 @@
 --- CHANGELOG ---
+--- Future ---
+    * Added method ResultDymolaBinary.get_variables_data. Included some minor refactorization.
```

Comment:
Could maybe add a few words on what it does?

Reply:
Updated